src/main/java/oshi/software/os/OperatingSystem.java

```diff
@@ -48,4 +48,18 @@
      * @return Version.
      */
     OperatingSystemVersion getVersion();
+
+    /**
+     * Open file descriptors.
+     *
+     * @return long
+     */
+    long getOpenDescriptors();

```

Comment:
This method should probably be `getOpenFileDescriptors()` and I think it would be better placed in the `FileSystem` interface, rather than the `OperatingSystem`.